### PR TITLE
Feat: Reorganize Wallet PopUp

### LINKED ISSUE #932: Reorganized wallet pop-up to show uninstalled wallets last
Capture d’écran 2024-11-07 à 11 24 36

OKX and Bitget wallets are not installed and should then be showed in the last position instead of right after argent/braavos

You can reuse the `sortConnectors` function [here](https://github.com/lfglabs-dev/app.starknet.id/blob/c15caf3281e3a6916fc5e5922edc719097a2f04d/utils/connectorWrapper.ts#L31)

### PR DESCRIPTION
**Pull request by Villarley**
Added the function sortConnectors and applied logic to display available connectors first, followed by unavailable ones.

# Pull Request type

- Feature

Resolves: #932 

### Summary of Changes

- Added the sortConnectors function to sort connectors based on availability, moving unavailable connectors (e.g., Okx Wallet, Bitget Wallet if not installed) to the end of the list.
- Updated the Navbar component to utilize the sorted connectors for a more intuitive wallet selection experience.
- Verified compatibility between @starknet-react/core and starknetkit.

feature
testnet

## Summary by CodeRabbit

- **New Features**
	- Introduced sorting functionality for connectors in the Navbar, enhancing the user experience by prioritizing available connectors.
  
- **Bug Fixes**
	- Improved the handling of connector availability, ensuring a more intuitive display order.

- **Tests**
	- Added a comprehensive test suite for the new sorting functionality to ensure reliability and correct behavior.

### CODE REVIEW

File: utils/sorts.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { Connector } from "starknetkit";
```
Comment (fricoben) [inline utils/sorts.ts]: rename the file `connectors.ts`, we'll reuse it later for connectors utils

File: utils/sorts.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  return [
+    ...available,
+    // Reorganized not available connectors
+    ...notAvailable.sort((a, b) => {
+      const order = ["okxwallet", "bitkeep"]; // desired order
+      return order.indexOf(a.id) - order.indexOf(b.id);
+    }),
+  ];
+};
```
Comment (fricoben) [inline utils/sorts.ts]: you need to create a connector.test.js function to test your util and make sur it covers all hedge cases here :)

**Review by fricoben — CHANGES_REQUESTED** [review]
Test your function :)

**Review by fricoben — APPROVED** [review]
lgtm
